stanford/cme/group_vars/all

```diff
@@ -27,6 +27,7 @@ EDXAPP_FEATURES_USE_CME_REGISTRATION: true
 EDXAPP_LMS_NGINX_PORT: 80
 EDXAPP_MKTG_URL_LINK_MAP:
   COURSES: 'https://med.stanford.edu/cme/learning-opportunities/online.html'
+  FAQ: 'https://stanfordonline.zendesk.com/hc/en-us'
```

Comment:
Should this be `https://support.zoho.com/portal/stanfordonlinecmesupport/home`?

Reply:
yes it should.